### PR TITLE
New X.520 attribute types

### PR DESCRIPTION
**Pull request by eabalea**
This contribution adds 4 attribute types that were added in 2012 and 2014 in X.520: organizationIdentifier, countryCode3c, countryCode3n, and dnsName.
organizationIdentifier is mandatory for some eIDAS-related certificates.

### CODE REVIEW

**Comment by levitte** [comment]
I can argue that this is a trivial fix, 'cause the manual labor is adding the entries in `crypto/objects/objects.txt`, the rest is a `make update` run.

If you agree re triviality, please add the following line in your commit message:

```
CLA: Trivial
```

**Comment by eabalea** [comment]
I agree it's trivial. Tried to change the commit message, don't know if it works. Anyway, I already signed the CLA.

**Comment by kroeckx** [comment]
I can't find a source for the 2.5.4.100 OID ...

**Comment by kroeckx** [comment]
It seems there are conflicts

**Comment by levitte** [comment]
@eabalea, the CLA you submitted is for `erwann@abalea.com`, which isn't the username you use in your commits (`erwann.abalea@docusign.com`)

**Comment by eabalea** [comment]
2.5.4.100 is listed in X.520 Corrigendum 1 (11/2014).

**Comment by kroeckx** [comment]
Oh, for some reason I missed it but found it now.

File: crypto/asn1/tbl_standard.h [hunk 1]
```diff
@@ -52,6 +52,9 @@ static const ASN1_STRING_TABLE tbl_standard[] = {
      STABLE_NO_MASK},
     {NID_INN, 1, 12, B_ASN1_NUMERICSTRING, STABLE_NO_MASK},
     {NID_OGRN, 1, 13, B_ASN1_NUMERICSTRING, STABLE_NO_MASK},
-    {NID_SNILS, 1, 11, B_ASN1_NUMERICSTRING, STABLE_NO_MASK}
+    {NID_SNILS, 1, 11, B_ASN1_NUMERICSTRING, STABLE_NO_MASK},
+    {NID_countryCode3c, 3, 3, B_ASN1_PRINTABLESTRING, STABLE_NO_MASK},
+    {NID_countryCode3n, 3, 3, B_ASN1_NUMERICSTRING, STABLE_NO_MASK},
+    {NID_dnsName, 0, -1, B_ASN1_UTF8STRING, STABLE_NO_MASK}
```
Comment (kroeckx) [inline crypto/asn1/tbl_standard.h]: I think that should be 1, -1

Comment (eabalea) [inline crypto/asn1/tbl_standard.h]: There's no size constraint for the DomainName type. One could argue that the "format of a (internationalized) domain name" implies a size constraint, and RFC1034 doesn't accept an empty domain.

**Comment by richsalz** [comment]
the actual submission is four lines in the header file and four in obj_mac.  Perhaps those are trivial changes.

**Comment by kroeckx** [comment]
@eabalea: You know we're waiting on you to fix the merge conflict?

**Comment by eabalea** [comment]
Sorry, done.

**Comment by mattcaswell** [comment]
This PR has merge commits which are not allowed. Please can you remove them (rebase as required)?

**Comment by eabalea** [comment]
I tried to rebase, but I'm not fluent enough in git and messed things up. Too much work for trivial changes, I'm dropping this PR for now. Sorry.

**Comment by richsalz** [comment]
No problem, I did it in #3590 which we just merged.

**Pull request closed without merging**